caterva2/services/sub.py

```diff
@@ -99,6 +99,9 @@ def __init__(self, abspath, path, metadata=None):
                 self._typesize = metadata.cparams.typesize
                 self._chunksize = metadata.chunksize
                 self._nbytes = metadata.nbytes
+                self._cparams = dict(metadata.cparams)
```

Comment:
In line with the note above, this can be moved into common code for SChunk and NDArray.

Reply:
Inavlid comment